homeassistant/components/recorder/statistics.py

```diff
@@ -464,7 +486,7 @@ def _compile_hourly_statistics(session: Session, start: datetime) -> None:
                 "mean": _mean,
                 "min": _min,
                 "max": _max,
-                "circular_mean": _circular_mean,
+                "circular_mean": adjust_circular_mean(engine, _circular_mean),
```

Comment:
If the stats list is long, doing a function call in the loop might get expensive